doc/source/notebooks/advanced/mcmc.pct.py

```diff
@@ -340,6 +340,10 @@ def plot_joint_marginals(samples, parameters_dict, y_axis_label):
 
 gpflow.utilities.print_summary(model)
 
+# %%
+# TODO confirm whether we should allow MCMC on a parameter without prior, add a prior, or set to non-trainable.
```

Comment:
SGPMC contains the inducing points Z. Should these be included in the MCMC sampling, despite not having a prior (implicit uniform prior)? Should they be excluded from MCMC (set as non-trainable)? Should we give them a prior (which one)? @jameshensman